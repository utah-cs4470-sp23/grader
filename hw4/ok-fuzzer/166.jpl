write image O to ""
read image "" to s
write image w (  ) to "R{"
show [ true ]
assert true, ""
time read image "/r" to y [ j ]
read image "" to s [ K , J , A ]
assert 5952., ""
show [ array [ f : sum [  ] 508 ] [ 19 ] ,  ]
time print "*|"
write image if sum [  ] T then array [  ] 8 else Y to ""
time time let H = [ 7.6 ,  ]
show 27
write image if true then sum [ P : false ] [  ] else { { Y , true } } to "j"
assert ( [ C ] ), "\"
let S = true { 795756 }
type F = bool
print "M"
time let { {  } , g } = {  }
assert [  ], "?r;"
time time let g = sum [ F : { true ,  } ] t ( 6205. )
print ""
let { {  } } = 0
write image true to ""
show c ( 5017. ) [ array [  ] { array [  ] ( true ) ,  } ]
time assert true, "x"
read image "<X" to C
type H = int [  ]
let {  } = array [  ] m (  ) [ false ]
assert { .04 }, "~"
assert {  }, ""
assert A ( ( true ) { 2 } ) { 45 }, ""
print "J>V"
assert [ { 607 } ,  ], ""
write image array [  ] {  } { 7 } to ""
show 9
time show array [ r : { 2 }, M : true ] array [ w : [ .97760 , 7 , I ,  ] ] ( {  } )
read image "" to G [ m , t , Q , G , w , F ]
assert l, ""
show p
let g [ H ] = ( 9 [ array [ U : sum [  ] false ] true ] ) [ true ]
type T = float
type x = { bool }
read image "&'" to d [ d ]
write image true { 84841 } to ""
time type f = int
let { X , T [ f , K , I ] } = .537
time read image "" to j
read image "" to S
let N [ t , X , U ] = ( 1 ) [ array [  ] .2078 ]
write image [ 35903. ] to ""
let {  } = [ true ]
