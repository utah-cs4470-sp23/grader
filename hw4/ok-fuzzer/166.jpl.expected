(WriteCmd (FalseExpr) "")
(FnCmd
 N
 ((TupleBinding (VarBinding (VarArgument tQ3) (VarType r_))))
 (ArrayType
  (TupleType
   (TupleType (BoolType) (FloatType))
   (ArrayType (VarType W2s.6p) 1)
   (ArrayType (TupleType) 5))
  2)
 (LetStmt (ArgLValue (ArrayArgument PT s_)) (ArrayLiteralExpr))
 (ReturnStmt (IntExpr 271987)))
(AssertCmd (VarExpr Ph_) "")
(WriteCmd (IntExpr 7) "")
(ShowCmd (FloatExpr 2))
(TimeCmd (TimeCmd (LetCmd (ArgLValue (ArrayArgument v. M.)) (TrueExpr))))
(FnCmd
 eidL
 ((TupleBinding)
  (TupleBinding
   (VarBinding (ArrayArgument B g d.) (IntType))
   (TupleBinding
    (VarBinding (ArrayArgument d.j) (ArrayType (IntType) 3))
    (VarBinding (ArrayArgument RQ32UM_W.9h7F89__0vm.40o) (IntType))))
  (TupleBinding))
 (IntType)
 (ReturnStmt (FalseExpr))
 (AssertStmt (FloatExpr 0) "")
 (ReturnStmt (TupleLiteralExpr)))
(WriteCmd (TrueExpr) "$")
(FnCmd
 l1Z8C42
 ((TupleBinding
   (TupleBinding
    (TupleBinding)
    (VarBinding (VarArgument S.x) (ArrayType (FloatType) 2))))
  (VarBinding (ArrayArgument r) (VarType N.))
  (TupleBinding)
  (VarBinding (VarArgument gPF8_6_2) (BoolType)))
 (ArrayType (VarType Dv) 1)
 (AssertStmt (VarExpr Vo) ""))
(FnCmd
 nW
 ((TupleBinding))
 (VarType o_3_)
 (ReturnStmt (FloatExpr 12))
 (LetStmt (ArgLValue (ArrayArgument u7)) (FalseExpr)))
(ReadCmd "$'L-" (VarArgument l14Iw5))
(TypeCmd Bx057_X (TupleType (TupleType)))
(ShowCmd (VarExpr w))
(PrintCmd "^.o\")
(AssertCmd (TupleIndexExpr (VarExpr H_1_) 94) "$")
(PrintCmd ".")
(FnCmd m ((TupleBinding) (TupleBinding)) (BoolType))
(TypeCmd p_5_3.u (FloatType))
